Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -4160,47 +4382,7 @@ void webkit_web_view_run_javascript_in_world(WebKitWebView* webView, const gchar
  */
 void webkit_web_view_run_async_javascript_function_in_world(WebKitWebView* webView, const gchar* body, GVariant* arguments, const char* worldName, GCancellable* cancellable, GAsyncReadyCallback callback, gpointer userData)
```

Comment:
You forgot to mark this one as deprecated in the doc comment.

Reply:
Good catch